styleguide/screens/Patterns/Tether/Tether.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, C, Note } from '../../../components/Scaffold/Scaffold';
+import Tether, {
+  VERTICAL_ATTACHMENTS,
+  HORIZONTAL_ATTACHMENTS,
+} from '../../../../components/Tether/Tether';
+import Select from '../../../../components/Form/Select/Select';
+import Option from '../../../../components/Form/Select/Option';
+import Checkbox from '../../../../components/Form/Checkbox/Checkbox';
+import { Label } from '../../../../components/Form/FormComponents';
+
+import m from '../../../../globals/modifiers.css';
+import css from './Tether.css';
+import scaffoldCss from '../../../components/Scaffold/Scaffold.css';
+
+const Target = () => <button>Anchor</button>;
```

Comment:
Any particular reason you decided to use a button for the target?

Reply:
Just based on the story we have already have for the `Tether` component, happy to change it.